opentaxii/taxii/exceptions.py

```diff
@@ -36,8 +36,8 @@ def __init__(self, message, **kwargs):
 
 class UnauthorizedStatus(StatusMessageException):
 
-    def __init__(self, **kwargs):
-        super(UnauthorizedStatus, self).__init__(ST_UNAUTHORIZED, **kwargs)
+    def __init__(self, status_type=ST_UNAUTHORIZED, **kwargs):
+        super(UnauthorizedStatus, self).__init__(status_type=status_type, **kwargs)
```

Comment:
seems to me like we should just remove this class if it's possible to pass status types other than "unauthorized"

Reply:
We can't just remove intermediate exceptions without breaking backward compatibility. What if someone doing `except UnauthorizedStatus:`?

I think, there is nothing bad in having `UnauthorizedStatus` with `status_type="NOT_FOUND"`, for example. `status_type` is about the response, exception type is about internal server handling.

Reply:
> We can't just remove intermediate exceptions without breaking backward compatibility. What if someone doing except UnauthorizedStatus:?

This is a good reason, although I think OpenTAXII should be provided as-is and not be used as a library.

Reply:
Good point. Moved into #150